[RUM] document `actionNameAttribute` usage (#11014)
DataDog/documentation: content/en/real_user_monitoring/browser/tracking_user_actions.md

  Enter a valid email address
</div>
```

> Starting with [version 2.16.0][3], with the `actionNameAttribute` initialization parameter, you can specify your own attribute that will be used to name the action. For example:

Reviewer: Should we mention that `data-dd-action-name` takes precedence over the custom attribute if both exists on the element?

Author: sure

  Enter a valid email address
</div>
```

> Starting with [version 2.16.0][3], with the `actionNameAttribute` initialization parameter, you can specify your own attribute that will be used to name the action. For example:

Reviewer 2: ```suggestion
Starting with [version 2.16.0][3], with the `actionNameAttribute` initialization parameter, you can specify your own attribute that is used to name the action. For example:
```
